src/components/Auth/Login.jsx

```diff
@@ -1,7 +1,8 @@
 import "./Auth.css";
 import { Link } from "react-router-dom";
-import { useAuth } from "../../context/auth-context";
+import { useAuth } from "../../context";
```

Comment:
Suggestion: you can use absolute import for better code readability.  Read: https://dev.to/emanuelone/how-to-use-absolute-path-in-reactjs-503i

Reply:
will read about this, thanks